src/filters/fs.rs

```diff
@@ -369,23 +369,21 @@ fn bytes_range(range: Option<Range>, max_len: u64) -> Result<(u64, u64), BadRang
     let ret = range
         .iter()
         .map(|(start, end)| {
-            let start = match start {
-                Bound::Unbounded => 0,
-                Bound::Included(s) => s,
-                Bound::Excluded(s) => s + 1,
-            };
-
-            let end = match end {
-                Bound::Unbounded => max_len,
-                Bound::Included(s) => {
+            let (start, end) = match (start, end) {
+                (Bound::Unbounded, Bound::Unbounded) => (0, max_len),
+                (Bound::Included(b), Bound::Included(e)) => {
                     // For the special case where s == the file size
```

Comment:
I guess the comment means "e instead of s", right?
But why not use `a` and `b` var names for the destructured tuples?

Reply:
yeah, should be `s` instead of `b` thanks! `s` and `e` match the `start` and `end` variables used on the `let` above